### PR TITLE
Websocket pack optimizing

### PR DESCRIPTION
**Pull request by asvetlov**
Use precompiled `struct.Struct(fmt).pack(...)` objects instead of `struct.pack(fmt, ...)` etc.

### CODE REVIEW

**Comment by ludovic-gasc** [comment]
+1

File: aiohttp/websocket.py [hunk 1]
```diff
@@ -30,6 +30,15 @@
 
 Message = collections.namedtuple('Message', ['tp', 'data', 'extra'])
 
+UNPACK_HEADER = Struct('!BB').unpack
```
Comment (kxepal) [inline aiohttp/websocket.py]: You can optimize it more by not using struct for trivial cases at all. Single byte is a such.

Comment (asvetlov) [inline aiohttp/websocket.py]: Fixed. Good point!

File: aiohttp/websocket.py [hunk 2]
```diff
@@ -30,6 +30,14 @@
 
 Message = collections.namedtuple('Message', ['tp', 'data', 'extra'])
 
+UNPACK_LEN2 = Struct('!H').unpack_from
+UNPACK_LEN3 = Struct('!Q').unpack_from
+UNPACK_CLOSE_CODE = Struct('!H').unpack
+PACK_LEN1 = Struct('!BB').pack
```
Comment (kxepal) [inline aiohttp/websocket.py:36]: Here is the same case (;

Comment (kxepal) [inline aiohttp/websocket.py:36]: Sorry, nvm. I didn't check carefully how it been used below. Seems it's fine.

**Comment by kxepal** [comment]
+1 LGFM

**Pull request merged**